packages/zone/test/test-make-once.js

```diff
@@ -30,35 +30,29 @@ test.before(t => {
 /**
  * @param {import('ava').Assertions} t
  * @param {Zone} rootZone
- * @param {boolean} [heapOnlyZone]
  */
-const testOnce = (t, rootZone, heapOnlyZone = false) => {
+const testOnce = (t, rootZone) => {
```

Comment:
I take it as a positive symptom that in this PR the test cases be more regular between the heap vs virtual and durable.

Reply:
I like the idea of a `testOnce` for testing behavior that should be in common across zones. But these tests are a bit thin.

Reply:
> these tests are a bit thin.

Yeah, `testOnce` is really only meant to test the `makeOnce` behaviour.  I still need to make more comprehensive tests (elsewhere) that explore more of the API.